tssc/factory.py

```diff
@@ -81,17 +90,24 @@ def register_step_implementer(implementer_class, is_default=False):
             _IS_DEFAULT_KEY: is_default
         }
 
-    def run_step(self, step_name, step_config_runtime_overrides=None, environment=None): # pylint: disable=too-many-branches
+    @property
+    def config(self):
+        """
+        Returns
+        -------
+        TSSCConfig
+            Configuration used by this factory.
+        """
+        return self.__config
+
+    def run_step(self, step_name, environment=None): # pylint: disable=too-many-branches
```

Comment:
You're just disabling the branching warning? How many were there...?

Reply:
apparently not actually still needed. that function got refactored a few times.

Reply:
pushed new change removing the `disable`